app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -998,6 +999,20 @@ protected void setColorFromIcon() {
         }
     }
 
+    protected void setThumbnailImage(Bitmap bitmap) {
```

Comment:
Not sure if it's worth using `@Nullable` here?

```suggestion
    protected void setThumbnailImage(@Nullable Bitmap bitmap) {
```